Services/EnergyCostAlgorithm.cs

```diff
@@ -0,0 +1,30 @@
+
+namespace Ultra_Saver;
+
+public class EnergyCostAlgorithm
+{
+    private static double totalEnergy = 0;
+    private static double totalEnergyCost = 0;
+    private static float costPerkWh = 0;
+
+    // powerScale is a proportion of power used for the specified time; scales 1 to 10; ex.: medium heat, 3/10 of power
+    private void electricPower (int appliencePower, short powerScale, int timeInMin){
+        totalEnergy += appliencePower * powerScale * timeInMin / 60_000;
```

Comment:
If we have a 1000W appliance that's always working at full power (10) then we should get 1kWh if it works for 60 mins.
With this calculation we get 1000W * 10 * 60min / 60_000 = 10, which is strange.
In my opinion powerScale should be a float, which represents how much power the appliance is using. For example a 1000W appliance working at 3/10 powerScale would be basically a 300W appliance. With this change everything seems to work perfectly.

Reply:
thank you for noticing this mistake.